### DISCUSSION TITLE
JQuery 1.2 < 3.5.0 Multiple XSS vulnerability

### DISCUSSION THREAD

**Discussion by akash0361** [community/community#31779] [Other Feature Feedback, Questions, & Ideas]
Gogs installed version is 0.12.10

Getting JQuery vulnerability of JQuery

What needs to be done to remediate it?

Please suggests ASAP.

**Comment by queenofcorgis** [comment]
Hi @akash0361,

Thanks for being a part of the GitHub Community! We're glad you're here.

I wanted to jump in and mention that if you're looking for help for this specific topic, you might want to try asking for help somewhere that focuses on https://jquery.com/. It's definitely possible that another GitHub user might have run into this same issue and can help, but the GitHub Community on Discussions focuses primarily on topics related to GitHub itself or collaboration on project development and ideas. We want to make sure you’re getting the best support you can, but this space may not be the right place for this particular topic.

Best of luck!
